applications/queryWeb3.md

```diff
@@ -0,0 +1,292 @@
+
+- User Roles: Polkadot users including investors, operators, data analysts, technical personnel, and media personnel.
+
+![Yield query](https://github.com/Query-Web3/product/blob/main/UI/Yield%20query.webp)
+
+Use Case 3:Vol & Txns Query
+
+- Business Description：allows users to query transaction volume, transaction count, and token trading volume data for any chain within the Polkadot ecosystem over a specified time period. Users can analyze this data across various time cycles, such as daily, weekly, monthly, quarterly, semi-annual, and annual cycles. The query results will include year-over-year (YoY) and month-over-month (MoM) metrics and so on .Additionally, based on these query results, our platform can generate reports showing the top 10 chains by transaction volume (or transaction count, YoY, MoM、QoQ) for a specific period, as well as trend line charts and other visual representations over a given time frame. This enables users to understand the current state and development trends of each chain within the Polkadot ecosystem. For example, users can observe whether the monthly transaction volume or transaction count has increased or decreased over the past year. This information helps users understand the status and trends of various chains within the Polkadot ecosystem, identify chains with investment potential or risks, and make informed investment and operational decisions.
+
+- data process: Initially, extract data from APIs like Subscan, Hydration, Bifrost and so on for DOT, KSM, Hydration, and Bifrost, process it into multidimensional BI data, and save it to MySQL to form the data source.
+
+- Business folw ： the same as use case2
+
+- **User Roles**: Polkadot users including investors, operators, data analysts, technical personnel, and media personnel.
+
+![Vol&Txns query](https://github.com/Query-Web3/product/blob/main/UI/vol%26txns%20query.webp)
+
+
+### Ecosystem Fit
+
+Help us locate your project in the Polkadot/Substrate/Kusama landscape and what problems it tries to solve by answering each of these questions:
+
+- Where and how does your project fit into the ecosystem?
+
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+Our target audience includes parachain/dapp/wallet/UI developers, designers, our own user base, and users of other dapps within the ecosystem.
+
+- What need(s) does your project meet?
+QueryWeb3 meets the need for timely and integrated information from the Polkadot ecosystem, addressing issues such as finding high-yield financial products, getting timely feedback on information queries, tracking major wallet movements, and identifying airdrops and NFT rewards.
+
+- How did you identify these needs? Please provide evidence in the form of (scientific) articles, forum discussions, case studies, or raw data.
```

Comment:
Feel free to add a short answer here as well.

Reply:
I have been closely following and studying the Polkadot ecosystem for the past few years and have been an early investor in Polkadot. Based on my experience, I have encountered some pain points within the Polkadot ecosystem, which have prompted me to create a product to address these issues. Here are the details:

**Pain points**

1、How to find the highest yielding financial products at present? Which chains are they on?
2、If we raise an information question in the Polkadot ecosystem, can we get timely and effective feedback?
3、What are the major movements in the Polkadot ecosystem wallets? What have they done?
4、What activities are there in each of the Polkadot and kusama ecosystem projects?
5、Can we still send some NFT benefits to target users?
6、What airdrops are there in the Polkadot ecosystem projects? Which airdrops can I participate in?
7、Based on the Polkadot and Kusama ecosystem, which projects are in the top 10 for on-chain transaction volume and transaction amount on a weekly, monthly, quarterly, semi-annual, and annual basis? Additionally, what are their year-over-year and month-over-month data?

**Solution**
1、Query the yield rates of financial products in various projects in the Polkadot ecosystem based on keyword conditions.
2、Analyze the data of any address in the Polkadot ecosystem including wallet data analysis 、 data dashboard and so on.
3、The abnormal situation of large whale transactions in the Polkadot ecosystem recently
4、Ranking of important financings in the Polkadot ecosystem.
5、NFT and peripheral merchandise mall
6、Customized airdrop solution for many polkadot projects in future.
7、AI powered information service.
8、Provide equivalent data for on-chain transaction volume and transaction amount on a weekly, monthly, quarterly, semi-annual, and annual basis, as well as year-over-year and month-over-month growth data.